### DISCUSSION TITLE
有需要讨论的研究问题，请转到 X-lab Discussions 上讨论

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-research#74] [Announcements]
Goto：https://github.com/orgs/X-lab2017/discussions

**Comment by tyn1998** [comment]
王老师，这是说open-research的Discussion暂时不用吗？如果是这样的话直接可以在仓库的Settings里关掉就行。

**Reply by will-ww** [reply]
> 王老师，这是说open-research的Discussion暂时不用吗？如果是这样的话直接可以在仓库的Settings里关掉就行。

嗯，不用了，我们集中到一个论坛上，关掉也行，放到这里给大家一个指引~

**Reply by tyn1998** [reply]
OK~
